### PR TITLE
docs: add execution policy warning message Closes #44598

### LINKED ISSUE #44598: Windows execution policy not mentioned in docs
### Description

The issue is described [here](https://github.com/angular/angular-cli/issues/22256#issuecomment-1003678055) 

You cannot possibly be held responsible for all contingencies for every system, that would be unreasonable. However, it would benefit the community, especially those of newcomers, were this issue at least mentioned as known in your docs. A timestamped solution would be most advantageous.

Whether or not this is normal behaviour is irrelevant, most other projects which encounter these issues make mention of the fact. The reality is that this particular problem is not documented and has fallen between the cracks, rendering your getting started doc unhelpful as completing it is left up to chance for the uninitiated. It is cost-free to add this and would speed up the education of those learning the framework. 

It serves no-one and nothing to omit this problem and solution from your docs and it should be noted there are 5 or 6 disparate solutions listed on SO, where only one is necessary.

More importantly, these solutions vary in their awareness and consideration of security measures. This alone makes it necessary to guide the user based on your framework's limited requirements.

### What is the affected URL?

_No response_

### Please provide the steps to reproduce the issue

On a fresh Windows 10 install, your getting started examples do not work do to limitations on the Execution Policy.

### Please provide the expected behavior vs the actual behavior you encountered

Your starting examples should provide adequate information for execution, including minimal permissions required on a system-by-system basis. 

### Please provide a screenshot if possible

_No response_

### Please provide the exception or error you saw

_No response_

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

_No response_

### PR DESCRIPTION
**Pull request by yousefalaqra**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [DONE ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ DONE] Tests for the changes have been added (for bug fixes/features)
- [ DONE] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
Adding a warning info message for windows users in the setup guide in order for them to make sure that they have enabled the execution policy already.

- [x] Documentation content changes

## What is the current behavior?
In the setup guide, there's no information for windows users to inform them about the execution policy

Issue Number: 44598

## What is the new behavior?
The setup guide now has a piece of information that tells Windows users about the execution policy 

## Does this PR introduce a breaking change?
-  No

### CODE REVIEW

**Comment by yousefalaqra** [comment]
@aikidave Could you please re-scan the pr, I've pushed the first commit within the not licensed user.

File: aio/content/guide/setup-local.md [hunk 1]
```diff
@@ -62,6 +61,13 @@ To install the Angular CLI, open a terminal window and run the following command
   npm install -g @angular/cli<aio-angular-dist-tag class="pln"></aio-angular-dist-tag>
 </code-example>
 
+<div class="alert is-helpful">
+ <p>Sets the PowerShell execution policies for Windows computers:</p>
```
Comment (aikidave) [inline aio/content/guide/setup-local.md:65]:
```suggestion
 <p>To set the PowerShell execution policies for Windows computers:</p>
```

To comply with style guides.

**Review by aikithoughts — COMMENTED** [review]
Minor change, otherwise LGTM!

File: aio/content/guide/setup-local.md [hunk 2]
```diff
@@ -62,6 +61,13 @@ To install the Angular CLI, open a terminal window and run the following command
...
+ <code-example language="sh">
+  Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned<aio-angular-dist-tag class="pln"></aio-angular-dist-tag>
```
[the remark this answers is not in the document]
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md:67]: @MrJithil 
Based on my personal experience, I think  `Unrestricted`.   
Should I change it?

File: aio/content/guide/setup-local.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Setting up the local environment and workspace
 
-This guide explains how to set up your environment for Angular development using the [Angular CLI tool](cli "CLI command reference").
+This guide explains how to set up your environment for Angular development using the [Angular CLI tool](cli 'CLI command reference').
```
Comment (JoostK) [inline aio/content/guide/setup-local.md:3]: Could you please undo all unrelated formatting changes throughout this PR (quotes, newlines, list bullets)

File: aio/content/guide/setup-local.md [hunk 4]
```diff
@@ -62,6 +61,13 @@ To install the Angular CLI, open a terminal window and run the following command
   npm install -g @angular/cli<aio-angular-dist-tag class="pln"></aio-angular-dist-tag>
 </code-example>
 
+<div class="alert is-helpful">
+ <p>To Set the PowerShell execution policies for Windows computers:</p>
+ <code-example language="sh">
+  Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy RemoteSigned<aio-angular-dist-tag class="pln"></aio-angular-dist-tag>
```
Comment (JoostK) [inline aio/content/guide/setup-local.md:67]: Using `<aio-angular-dist-tag>` is incorrect here.

File: aio/content/guide/setup-local.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MrJithil) [reply aio/content/guide/setup-local.md:67]: @aikidave may be the person who can confirm it.

File: aio/content/guide/setup-local.md [hunk 5]
```diff
@@ -62,6 +61,13 @@ To install the Angular CLI, open a terminal window and run the following command
   npm install -g @angular/cli<aio-angular-dist-tag class="pln"></aio-angular-dist-tag>
 </code-example>
 
+<div class="alert is-helpful">
+ <p>To Set the PowerShell execution policies for Windows computers:</p>
```
Comment (JoostK) [inline aio/content/guide/setup-local.md:65]: I have never needed to do this on Windows, so this may not be needed in all cases. This may need some introduction as to _when_ this is needed together with a link to Microsoft's docs. Additionally, can people run into security restrictions when running this command, causing it to fail?

`To Set` needs to become `To set`.

**Review by JoostK — COMMENTED** [review]
Please squash all commits using a `docs: add execution policy warning message` as commit message. The `aio` scope should not be used, as `aio` is concerning the angular.io application itself, whereas this change is just targeting the docs content.

Please also include a `Closes #44598` at the end of the commit message.

File: aio/content/guide/setup-local.md (hunk 3 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md:3]: Done

File: aio/content/guide/setup-local.md (hunk 4 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: Removed!

File: aio/content/guide/setup-local.md (hunk 5 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: Done

**Comment by yousefalaqra** [comment]
@JoostK  I'm unable to squash the commits locally, do you have a reference in order to showcase how to handle that?

**Comment by JoostK** [comment]
You'll need to rebase interactively to accomplish that. There are [some instructions here](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-rebase).

**Review by aikithoughts — APPROVED** [review]
LGTM! Thank you!

Reviewed-for: global-docs-approvers

**Comment by aikidave** [comment]
@yousefalaqra If you need help with the rebase, please ping me! I'd be glad to help!

**Comment by yousefalaqra** [comment]
> @yousefalaqra If you need help with the rebase, please ping me! I'd be glad to help!

Rebase is completed, all the commits were squshed into one with the correct message successfully @JoostK

File: aio/content/guide/setup-local.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: @aikidave  What do you think? should resolve this or change it to Unrestricted?

File: aio/content/guide/setup-local.md (hunk 3 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: @JoostK could you please check the latest commit and resolve the comment?

File: aio/content/guide/setup-local.md (hunk 4 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: @JoostK  I've removed `<aio-angular-dist-tag>` component, could you please resolve the comment?

File: aio/content/guide/setup-local.md (hunk 5 above)
Comment (yousefalaqra) [reply aio/content/guide/setup-local.md]: @JoostK Could you please double-check it and resolve the comment?

**Comment by alxhub** [comment]
I think the rebase here is faulty, there seem to be a lot of unrelated changes. Can you take a look again, @yousefalaqra?

**Comment by yousefalaqra** [comment]
> I think the rebase here is faulty, there seem to be a lot of unrelated changes. Can you take a look again, @yousefalaqra?

@alxhub yes, of course. I'll take a look, but could you please point out the issue and share with me more details?

**Comment by JoostK** [comment]
The single commit changes 194 files, instead of just the doc. You seem to have rebased and squased on top of an incorrect base commit (or something like that, it's hard to say what happened exactly)

**Comment by AndrewKushnir** [comment]
As it was mentioned above, it looks like there was a rebase issue that resulted in 194 changed files in this PR. I'm closing this PR for now (due to the lack of response). Please reopen (or create a new PR) when you get a chance to look into it.
